WordPress/src/main/java/org/wordpress/android/util/SiteUtils.java

```diff
@@ -57,4 +58,32 @@ public static String getSiteIconUrl(SiteModel site, int size) {
 
         return siteIDs;
     }
+
+    /**
+     * Checks if site Jetpack version is higher than limit version
+     * @param site
+     * @param limitVersion minimal acceptable Jetpack version
+     * @return
+     */
+    public static boolean checkMinimalJetpackVersion(SiteModel site, String limitVersion) {
+        String jetpackVersion = site.getJetpackVersion();
+        if (site.isUsingWpComRestApi() && site.isJetpackConnected() && !TextUtils.isEmpty(jetpackVersion)) {
+            try {
+                // strip any trailing "-beta" or "-alpha" from the version
+                int index = jetpackVersion.lastIndexOf("-");
+                if (index > 0) {
+                    jetpackVersion = jetpackVersion.substring(0, index);
+                }
+                Version siteJetpackVersion = new Version(jetpackVersion);
+                Version minVersion = new Version(limitVersion);
+                return siteJetpackVersion.compareTo(minVersion) >= 0; // if the site has Jetpack 5.8 or newer installed
```

Comment:
Nitpick: Let's update the comment here, just to avoid confusion.

Reply:
I removed the comment so it didn't really make sense in a generic way (the code itself is really self-explanatory)